internal/nonkube/bootstrap/install.go

```diff
@@ -16,5 +39,261 @@ func Install(platform string) error {
+	}
+	err = cli.ContainerStart(sysControllerContainer.Name)
+	if err != nil {
+		return fmt.Errorf("failed to start system-controller container: %v", err)
+	}
+
+	err = createSystemdService(sysControllerContainer, platform)
+	if err != nil {
+		return fmt.Errorf("failed to create system-controller systemd service: %v", err)
+	}
+
+	fmt.Printf("Platform %s is now configured for Skupper\n", platform)
+
 	return nil
 }
+
+func configEnvVariables(platform string) (*ControllerConfig, error) {
+
+	controllerConfig := ControllerConfig{}
+	var err error
+
+	// Set CONTAINER_ENGINE
+	switch platform {
+	case "docker":
+		_, err = exec.LookPath("docker")
+		if err != nil {
+			return nil, fmt.Errorf("docker not found")
+		}
+
+	default:
+		_, err = exec.LookPath("podman")
+		if err != nil {
+			return nil, fmt.Errorf("podman not found")
+		}
+
+	}
+	err = os.Setenv("CONTAINER_ENGINE", platform)
+	if err != nil {
+		return nil, err
+	}
+
+	controllerConfig.containerEngine = platform
+
+	// Get current username
+	currentUser, err := user.Current()
+	if err != nil {
+		return nil, fmt.Errorf("Failed to get current user: %v", err)
+	}
+	uid := currentUser.Uid
+	controllerConfig.username = currentUser.Username
+
+	xdgRuntimeDir := os.Getenv("XDG_RUNTIME_DIR")
+	if xdgRuntimeDir == "" {
+		xdgRuntimeDir = fmt.Sprintf("/run/user/%s", uid)
+	}
+
+	containerEndpointDefault := fmt.Sprintf("unix://%s/podman/podman.sock", xdgRuntimeDir)
+
+	if platform == "docker" {
+		containerEndpointDefault = "unix:///run/docker.sock"
+	}
+
+	uidInt, _ := strconv.Atoi(uid)
+	xdgDataHome := "/output"
+	hostDataHome := api.GetHostDataHome()
+	if uidInt == 0 {
+		if platform == "podman" {
+			containerEndpointDefault = "unix:///run/podman/podman.sock"
+		}
+		hostDataHome = "/var/lib/skupper"
+	}
+
+	if err := os.MkdirAll(api.GetDefaultOutputNamespacesPath(), 0755); err != nil {
+		return nil, fmt.Errorf("Failed to create directory: %v", err)
+	}
+
+	containerEndpoint := os.Getenv("CONTAINER_ENDPOINT")
+	if containerEndpoint == "" {
+		containerEndpoint = containerEndpointDefault
+	}
+
+	err = os.Setenv("CONTAINER_ENDPOINT_DEFAULT", containerEndpointDefault)
+	if err != nil {
+		return nil, err
+	}
+
+	controllerConfig.containerEndpointDefault = containerEndpointDefault
+
+	err = os.Setenv("SKUPPER_OUTPUT_PATH", hostDataHome)
+	if err != nil {
+		return nil, err
+	}
+
+	controllerConfig.hostDataHome = hostDataHome
+
+	err = os.Setenv("XDG_DATA_HOME", xdgDataHome)
+	if err != nil {
+		return nil, err
+	}
+
+	controllerConfig.xdgDataHome = xdgDataHome
+
+	err = os.Setenv("CONTAINER_ENDPOINT", containerEndpoint)
+	if err != nil {
+		return nil, err
+	}
+
+	controllerConfig.containerEndpoint = containerEndpoint
+
+	return &controllerConfig, nil
+}
+
+func createSystemdService(container container.Container, platform string) error {
+
+	// Creating startup scripts
+	startupArgs := controller.StartupScriptsArgs{
+		Name:     container.Name,
```

Comment:
My thought is that we could name the service as `skupper-controller` (keeping container name as is).

Reply:
done